.github/workflows/Pack_Release.yml

```
@@ -0,0 +1,131 @@
+          # 清理工作目录
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}-lite" -Recurse -Verbose
+
+      - name: 构建打包版与安装程序版
+        # 顺便为安装程序版一起打包，打包完复制一份
+        env:
+          PYTHONIOENCODING: utf-8
+        run: |
+          Copy-Item -Path "src" -Destination "Release/Fufu_Tools.v${{ github.event.inputs.version }}" -Recurse -Verbose
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/芙芙工具箱.pyw" -Verbose
+          Rename-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/芙芙工具箱(for打包).pyw" -NewName "芙芙工具箱.pyw" -Verbose
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/Tools/【实验性工具】/hosts修改/调用.py" -Verbose
+          Rename-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/Tools/【实验性工具】/hosts修改/调用(For-打包版).py" -NewName "调用.py" -Verbose
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/Tools/休息一下/自启动设置.py" -Verbose
+          Rename-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/Tools/休息一下/自启动设置（For-打包版）.py" -NewName "自启动设置.py" -Verbose
+
+          python DEV-PACK.py "Release/Fufu_Tools.v${{ github.event.inputs.version }}" "src/ico.ico" "Release/logs/pack-exe" -s
+
+          # 分离安装程序版
+          Copy-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}" -Destination "Release/Fufu_Tools.v${{ github.event.inputs.version }}-exe" -Recurse -Verbose
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}/芙芙工具箱(for安装程序).exe" -Verbose
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}-exe/芙芙工具箱.exe" -Verbose
+          Rename-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}-exe/芙芙工具箱(for安装程序).exe" -NewName "芙芙工具箱.exe" -Verbose
+
+          # 更新版本号 - 分离完后再更新版本号，如果后续有修改需要打包的文件的版本号，请在运行 DEV-PACK.py 之前更新版本号
+          python 自动化脚本/修改版本号.py "${{ github.event.inputs.version }}" "pack" "Release/Fufu_Tools.v${{ github.event.inputs.version }}"
+          python 自动化脚本/修改版本号.py "${{ github.event.inputs.version }}" "exe" "Release/Fufu_Tools.v${{ github.event.inputs.version }}-exe"
+
+          # 压缩为 zip - 打包版
+          7z a -tzip "Release/Fufu_Tools.v${{ github.event.inputs.version }}.zip" "Release/Fufu_Tools.v${{ github.event.inputs.version }}/*"
+          # 压缩为 7z - 打包版
+          7z a -t7z "Release/Fufu_Tools.v${{ github.event.inputs.version }}.7z" "Release/Fufu_Tools.v${{ github.event.inputs.version }}/*"
+          # 压缩为 7z - 压缩等级 9 - 极限压缩 - 打包版
+          7z a -t7z -mx9 "Release/Fufu_Tools.v${{ github.event.inputs.version }}-Extreme_compression.7z" "Release/Fufu_Tools.v${{ github.event.inputs.version }}/*"
+          # 清理工作目录 - 打包版
+          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}" -Recurse -Verbose
+
+          # 编译安装程序 - 安装程序版
+          iscc "Installer/Installer.iss"
```

```suggestion
          iscc "Installer/Installer.iss"
          # 清理工作目录 - 安装程序版
          Remove-Item -Path "Release/Fufu_Tools.v${{ github.event.inputs.version }}-exe" -Recurse -Verbose
```

我认为 iss 文件存在问题，但我找不到证据。

---

那就等你找到证据了再 requested changes